tests/testTriggers.go

```diff
@@ -378,6 +378,10 @@ const (
 	MetadataPoolResizeDisk = "metadatapoolResizeDisk"
 	// PoolAddDisk resize storage pool using add-disk
 	PoolAddDisk = "poolAddDisk"
+	// PoolAuto resize storage pool using auto
+	PoolAuto = "poolAutoDisk"
```

Comment:
can we consider keeping it as poolAuto instead of poolAutoDisk ?

Reply:
My bad. Will correct it.

Reply:
Done.

Reply:
Added Expansion prefix.